frontend/webapp/hooks/actions/useActionFormData.ts

```diff
@@ -29,6 +29,7 @@ export function useActionFormData() {
 
   const resetFormData = () => {
     setFormData({ ...INITIAL });
+    setFormErrors({});
   };
 
   const validateForm = (params?: { withAlert?: boolean; alertTitle?: string }) => {
```

Comment:
nit: create useGenericForm for share dup code between the forms hooks

Reply:
created a notion task :)